apps/core/graphql/schema.ex

```diff
@@ -6,6 +6,8 @@ defmodule Core.GraphQL.Schema do
   import_types Core.GraphQL.Types.AuthMutations
   import_types Core.GraphQL.Types.Viewer
 
+  import_types Core.GraphQL.Users.Types
```

Comment:
Please make this `Core.GraphQL.Types.User` to be consistent with all the `Core.GraphQL` stuff.